en/docs/includes/integration/PULL-CONTENT-migration-esb-mi.md

```diff
@@ -7,14 +7,14 @@ Listed below are some of the advantages of moving to API-M 4.0.0 from the ESB.
 	!!! Tip
 		WSO2 ESB 5.0, the ESB profile of WSO2 EI 6.x, the Micro Integrator of WSO2 EI 7.x, as well as the Micro Integrator of WSO2 API-M 4.0.0 contains versions of the same WSO2 ESB runtine. 
```

Comment:
The page name itself may be an issue coz it has upper case characters in it.
PULL-CONTENT-migration-esb-mi.md